Another zombies rework (#637)
space-wizards/docs: src/en/space-station-14/round-flow/proposals/zeds rework.md

## Background

We currently have zombies on Wizden, most downstreams have zombies and a lot of SS13 servers had some sort of implementation of zombies. This game mode has received mostly good player feedback and is rather well liked, but still lacks a design doc, and has some rough edges. This document seeks to describe how zombies should develop in the future: a zombie round should reverse the usual hunter-hunted relationship of antagonists and crew members as the last survivors desperately try to fend off the infected in the best zombie horror movie fashion.

## Features

At the beginning of a Zombies round, the infection will be spread to a few random patients zeroes. It should then slowly spread, disrupting station activities, until the remaining survivors must go into hiding, build barricades and arm themselves: the station shouldn’t be able to easily ignore the zombie virus.

> The infection spreads through exchange of bodily fluids; of course, this means that infected individuals will pass on the infection through bites (as it should be and is in most zombie media). Due to how combat works in SS14, having a bite infect a player 100% of the time would feel rather bad: despite zombies infecting on bites, the crew should still be somewhat able (either trough protection or luck) to fend of zombies using melee weapons.

Reviewer: ```suggestion
The infection spreads through exchange of bodily fluids; of course, this means that infected individuals will pass on the infection through bites (as it should be and is in most zombie media). Due to how combat works in SS14, having a bite infect a player 100% of the time would feel rather bad: despite zombies infecting on bites, the crew should still be somewhat able (either through protection or luck) to fend of zombies using melee weapons.
```

> The infection spreads through exchange of bodily fluids; of course, this means that infected individuals will pass on the infection through bites (as it should be and is in most zombie media). Due to how combat works in SS14, having a bite infect a player 100% of the time would feel rather bad: despite zombies infecting on bites, the crew should still be somewhat able (either trough protection or luck) to fend of zombies using melee weapons.

Reviewer: You should explain why having a bite infect a player 100% of the time is bad, rather than saying "how combat works"

> Once infected, players will be granted a relatively short grace period before displaying symptoms (we should maybe wait for some sort of virology before introducing visible symptoms) and eventually turning. It should be possible for crew members to tell infected-but-not-zombified-yet individuals from sane crew members. The infection should zombify fast enough as for it to be an actual danger during a round.

Reviewer: ```suggestion
Once infected, players will be granted a relatively short grace period before displaying symptoms (we should maybe wait for some sort of virology before introducing visible symptoms) and eventually turning. It should be possible for crew members to tell individuals infected with the virus apart from non-infected crew members. The infection should zombify fast enough as for it to be an actual danger during a round.
```

> Upon turning, players lose basic abilities such as talking and operating machinery. They pretty much get reduced to biting machines who must now roam the halls of the station to infect more individuals.

Reviewer: you should just specify that they lose their hands, but also zombies with hands should have innate prying

Newly-infected zombies should still be entertained by the game; we don’t want them ghosting. We should thus introduce classic zombie games tropes such as:
 - Mutant zombies (fast, pouncer, brute, etc) each with their strengths and weaknesses (brute zombies are way stronger but can’t infect living individuals, fast zombies are extremely weak to brute, etc). Zombies should still work better in hordes, though. We don’t want a magic invincible berserker zombie that can run through crowds of survivors and murder & infect them all. Each mutation should be designed to either assist other zombies or to work well in hordes.

>  - Survivor detection / pinpointer ability to detect remaining survivors when only a few are left. Stragglers hiding in lockers or in secluded areas should not be safe from the infected. Having each and every (player-controlled) zombie suddenly home in on the remaining survivors would be absolutely terrifying.

Reviewer: ```suggestion
 - Survivor detection / pinpointer ability to detect remaining survivors when only a few are left. Stragglers hiding in lockers or in secluded areas should not be safe from the infected. Having each and every (player-controlled) zombie suddenly home in on the remaining survivors would prevent round-stalling, aswell as add to the remaining survivors versus horde feeling mentioned before.
```

> One way to implement mutations would be trough giving different initial infected different strains of the virus; this would encourage all initial infected to infect the most individuals possible instead of having only one II turn and the other sabotage the station while quaffing dylovene (initial infected should *not* be able to stay unturned for the whole shift, so if this is not enough, then the poison damage from the infection could massively ramp up trough time).

Reviewer: ```suggestion
One way to implement mutations would be through giving different initial infected different strains of the virus; this would encourage all initial infected to infect the most individuals possible instead of having only one II turn and the other sabotage the station while quaffing dylovene (initial infected should *not* be able to stay unturned for the whole shift, so if this is not enough, then the poison damage from the infection could massively ramp up trough time).
```

> Another way for mutations to function would have them be completely random, and more common as the infection advances. Zombies could get a random mutation after a certain amount of time alive, with it being faster & less rare when more are infected.

Reviewer: ```suggestion
Another way for mutations to function would have them be completely random, and more common as the infection advances. Zombies could get a random mutation after a certain amount of time alive, with it being faster & less rare when more are infected, this would reward zombie players who don't ghost and stay active in the round.
```

The crew could thus build a somewhat zombie-proof zone through things like electrical fences, grids, barricades, etc. Of course, no area should ever be 100% safe from the horde. 

As mentioned before, zombies should also have *some* way to locate survivors when only a few stragglers are left alive; we don’t want them to wander aimlessly trough the station’s halls; while this makes sense from an external point of view, playing as a zombie without someone to bite is extremely boring and thus this should be avoided. They could thus get an innate nukeops-like pointer, basic directions toward survivors, "leader » zombies or something similar.

In general, zombies should get stronger and stronger the more the round advances, to prevent endless stalling by the crew. Mutations should be more common, stronger, zombies should do more damage or the infection could have a shorter incubation period. Of course, this should be balanced around the remaining number of cremwates vs the size of the infected horde. Think of it as rubber banding to constantly keep the round fun for both the zombies and the uninfected.

> Crew members should look for armor & clothes such as riot armor, biologic suits, winter coats, etc: with enough protection, you should be virtually immune to basic bite infections. You should be able to defend yourself from one zombie with a baseball bat and proper clothing without always getting infected.

Reviewer: ```suggestion
Crew members should look for armor & clothes such as riot armor, biosuits, winter coats, etc: with enough protection, you should be virtually immune to basic bite infections. You should be able to defend yourself from one zombie with a baseball bat and proper clothing without always getting infected.
```

> There should be some ways for crew members to know if another is infected or not; be it trough description symptoms (such as the character looking feverish) or random coughing, having someone on your team suddenly display those symptoms should give pressure the crew and make it so the zombie virus isn’t just a sudden transformation into a mean green biting machine with little to no symptoms beyond damage.

Reviewer: you should mention this earlier in the early game section, when talking about II symptoms and first infected

### Last stand

Once a certain percentage of station staff is infected, an evacuation shuttle should be automatically called and the round should devolve into a mad rush to secure evac, as both the crew and the infected horde need to make it to central command. This is where the round should climax, as every soul, living or not, on the station tries to make it on the shuttle.

> To ensure no one hides behind and to reinforce the "last stand" vibe, the escape pods could be disabled or made harder to access.

Reviewer: an overall issue with this document is that it seems like every round will play the exact same way, Zombies infect tons of people, last stand at evac with CBURN, with other gamemodes there's room for alternatives, Nukies going stealth, Syndicate doing Gimmicks, Revs stealthily converting command

At this point, a CBURN squad could be automatically dispatched by centcomm to defend the evacuation shuttle. This would have two benefits: 

 - It would provide interesting ghost roles for the many players who will get round-removed on zombie rounds (think of the poor zombified Pun Pun stuck in space, far from everyone, or the numerous zombies killed with no chance of ever being revived).
 - The military trying to sort infected individuals from those still healthy would make for great RP occasions (trying to hide the symptoms of the zombie virus as a simple fever, etc).

### In other game modes

> As for the nuclear operative’s romerol bundle, it simply needs to work like a real zombie game mode - it plays fairly well in the zombie apocalypse theme, since a frequent trope of this genra is biological warfare / semi-intentional medical disasters and nukies are pretty much doing exactly that. 

Reviewer: can you elaborate on what you mean by a real zombie game mode? Do you mean that romerol should include all the stuff mentioned in the doc like mutations

## Administrative & Server Rule Impact (if applicable)

> None, I hope :)

Reviewer: Worries about CBURN should be mentioned here, whats stopping CBURN from just kill all the survivors in fear that they're infected.
